apps/common-app/src/new_api/complicated/lock/index.tsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState } from 'react';
+import { View, Text, StyleSheet } from 'react-native';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  runOnJS,
+} from 'react-native-reanimated';
+import {
+  GestureDetector,
+  useCompetingGestures,
+  useLongPressGesture,
+  usePinchGesture,
+  useRotationGesture,
+  useSimultaneousGestures,
+  useTapGesture,
+} from 'react-native-gesture-handler';
+import { COLORS } from '../../../common';
+
+export default function Lock() {
+  const rotation = useSharedValue(-Math.PI / 2);
+  const savedRotation = useSharedValue(-Math.PI / 2);
+
+  const scale = useSharedValue(0.6);
+  const savedScale = useSharedValue(1);
+
+  const [locked, setLocked] = useState(true);
+  const snapThreshold = 0.4;
+  const scaleThreshold = 0.1;
+  const minScale = 0.5;
+  const maxScale = 1;
+  const TWO_PI = 2 * Math.PI;
+
+  // Tap to lock
+  const tap = useTapGesture({
+    onDeactivate: () => {
+      'worklet';
```

Comment:
If you use the latest `react-native-worklets`, this directive should be auto-included. Maybe it's worth it to update worklets and remove the directive?